media/js/mozorg/developer/css-grid-demo.js

```diff
@@ -0,0 +1,139 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+* License, v. 2.0. If a copy of the MPL was not distributed with this
+* file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+/* global Waypoint */
+/* eslint no-unused-vars: [2, { "varsIgnorePattern": "[wW]aypoint" }] */
+
+
+(function($) {
+    'use strict';
+
+    var $document = $(document);
+    var $html = $(document.documentElement);
+    var utils = Mozilla.Utils;
+
+    var $gridVideo = $('#grid-video');
+    var frameSrc = $gridVideo.data('frameSrc');
+
+    if (window.Mozilla.Client.isFirefox) {
+        if (window.Mozilla.Client.FirefoxVersion >= 52) {
```

Comment:
Good idea! We still need the basic "firefox/not-firefox" detection but if it's Firefox we can totally check for grid support instead of version number.